app/infra/keyGenService/keyGenService.js

```diff
@@ -14,7 +14,7 @@ class KeyGeneratorService {
     const seedAsUint8Array = Buffer.from(seed, 'hex');
     const left32BitsOfSeed = seedAsUint8Array.slice(0, nacl.sign.seedLength);
     const { publicKey, secretKey } = nacl.sign.keyPair.fromSeed(left32BitsOfSeed);
-    return { publicKey: Buffer.from(publicKey).toString('hex'), secretKey: Buffer.from(secretKey).toString('hex'), seed };
+    return { publicKey: Buffer.from(publicKey).toString('hex'), secretKey: Buffer.from(secretKey).toString('hex'), seed, resolvedMnemonic };
```

Comment:
rename to mnemonic

Reply:
This will clash with "mnemonic" param in wallet actions.

Reply:
you can rename it like this { mnemonic: returnedMnemonic }

Reply:
Renaming on destructuring still does not solve the issue. Unless I change the "mnemonic" parmam name in the saveNewWallet fuction to something else, like so: 
in keyGenService:
return { publicKey: Buffer.from(publicKey).toString('hex'), secretKey: Buffer.from(secretKey).toString('hex'), seed, **mnemonic: resolvedMnemonic** };

in wallet actions: 

<img width="862" alt="Screen Shot 2019-04-03 at 5 20 14 PM" src="https://user-images.githubusercontent.com/19342975/55486352-d5fe1f00-5634-11e9-816c-3a43ed12524f.png">

Reply:
this works just fine:
<img width="1172" alt="Screen Shot 2019-04-04 at 10 10 06" src="https://user-images.githubusercontent.com/16574879/55536205-d690c700-56c1-11e9-81b8-3e600209f699.png">
<img width="785" alt="Screen Shot 2019-04-04 at 10 09 27" src="https://user-images.githubusercontent.com/16574879/55536208-d8f32100-56c1-11e9-8b65-e8c76305f694.png">

What you've done - changed param name in function params destruct without changing param name in all usages of this function => broken app functionality

Reply:
OK np, this brings us back to how it was coded to begin with.

Reply:
it's not the same, the important thing is not breaking the app with untested changes